src/platform/SIMULATOR/sitl.c

```diff
@@ -646,6 +646,8 @@ bool motorPwmDevInit(motorDevice_t *device, const motorDevConfig_t *motorConfig,
     }
     device->vTable = &vTable;
     const uint8_t motorCount = device->count;
+    pwmMotorCount = device->count;
```

Comment:
Would it better to rename motorCount here (and other references in this file) ?
```suggestion
    pwmMotorCount = device->count;
```

Reply:
Like 

```bash
bool motorPwmDevInit(motorDevice_t *device, const motorDevConfig_t *motorConfig, uint16_t _idlePulse)
{
    UNUSED(motorConfig);

    if (!device) {
        return false;
    }
    device->vTable = &vTable;
    pwmMotorCount = device->count;
    printf("Initialized motor count %d\n", pwmMotorCount);
    pwmRawPkt.motorCount = pwmMotorCount;

    idlePulse = _idlePulse;

    for (int motorIndex = 0; motorIndex < MAX_SUPPORTED_MOTORS && motorIndex < pwmMotorCount; motorIndex++) {
        pwmMotors[motorIndex].enabled = true;
    }

    return true;
}
```

Reply:
yeah more in line with the code from pwm_output_hw.c of STM32 for example.